server/plugin/utils.go

```diff
@@ -332,3 +333,19 @@ func getToDoDisplayText(baseURL, title, url, notifType string) string {
 
 	return fmt.Sprintf("* %s %s %s\n", repoPart, notifType, titlePart)
 }
+
+// ItemExists check if the element exist in the passed array
+func ItemExists(arrayType interface{}, item interface{}) (bool, int) {
```

Comment:
This can accept a string and a slice of strings. No need for fancy reflection.

Reply:
resolved , made it simple